library/src/scripts/forms/select/SelectBox.tsx

```diff
@@ -34,6 +35,7 @@ interface IProps {
     openAsModal?: boolean;
     selectedIndex?: number;
     renderLeft?: boolean;
+    handleOnChange?: (...args: any[]) => any;
```

Comment:
I'd recommend naming this differently. Normally a prop doesn't have `handle` at the beginning of it. The hander would.

Eg.

```
<SelectBox onChange={handleChange} />
```

Also we are we using `any` here? This should have types for the arg array and for the return type (does it even have a return type? I feel like the return type is normally `void` for this type of thing).

Reply:
Made changes-
add onChange() props to "DropdownItemButton" in SelectBox component.
removed "LanguageDropdown" component.